CanonicalLayouts/list-detail-compose/app/src/main/java/com/example/listdetailcompose/ui/ListDetailSample.kt

```diff
@@ -204,12 +251,36 @@ private fun ListContent(
                     .then(interactionModifier)
                     .fillMaxWidth()
             ) {
-                Text(
-                    text = word,
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .padding(8.dp)
-                )
+                Row {
+                    val imageModifier =
+                        if (isSmallerThanExpanded && !isDetailVisible) {
+                            with(sharedTransitionScope) {
+                                val state = rememberSharedContentState(key = word.word)
+                                Modifier
+                                    .padding(horizontal = 8.dp)
```

Comment:
Nit: This `.padding` could be lifted out of the `if` by using `Modifier.then` so it isn't duplicated.

Reply:
Oh, thanks, I didn't think of "then", I'm changing it now